.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
+  -->
+### Summary of Problem
+
+
+### Steps to Reproduce
+
+**What behavior do you observe when encountering this issue?**
+
+**Please provide one of the following:**
+
+- Reduced program demonstrating issue *(preferred)*:
+- Original program demonstrating issue:
+- Description of how to recreate the issue *(last resort)*:
+
+**Please provide any of the following that are necessary to reproduce the issue:**
+
+- How to compile:
+- How to run:
+- Input files:
+- Helper modules:
```

Comment:
I agree this should be part of source code and am OK removing it. If it becomes a consistent misunderstanding in the future, we can re-introduce it.